addons/languages/english/pack/src/main/res/xml/english_autotext.xml

```diff
@@ -103,6 +104,7 @@
     <word src="isnt">isn't</word>
     <word src="isthe">is the</word>
     <word src="itd">it'd</word>
+    <word src="itslian">italian</word>
```

Comment:
why is that a common typo?

Reply:
the "s" and the "a" are next to each other. I make this typo often. But we can remove it.

Reply:
yeah, please remove

Reply:
done